torchaudio/transforms.py

```diff
@@ -1689,14 +1657,18 @@ class MVDR(torch.nn.Module):
     PSD matrices of speech and noise, respectively.
```

Comment:
You might want to also refer to a paper explaining MVDR?
If ESPNet is refactored in future, your link above may die, an arxiv one may be more permanent.
Although I suppose people can just google it.

Reply:
Okay, will add a citation to the MVDR paper after the ESPNet link.

Reply:
Addressed in https://github.com/pytorch/audio/pull/2181/commits/4c1a539aefaa9bb27e2f3628f694cf107386001d